tools/Tools.Common/Loaders/MetadataLoader.cs

```diff
@@ -30,17 +30,21 @@ namespace Tools.Common.Loaders
 {
     public class MetadataLoader
     {
+        private static string _rootPath = Path.GetFullPath(Path.Combine(Assembly.GetExecutingAssembly().Location, "..", "..", ".."));
```

Comment:
What is the difference between Assembly.GetExecutingAssembly().Location and AppDomain.CurrentDomain.BaseDirectory?

Reply:
Assembly.GetExecutingAssembly().Location is always azure-powershell\artifacts\VersionController\VersionController.Netcore.dll. AppDomain.CurrentDomain.BaseDirectory is the workspace of powershell.